src/Arcus.Scripting.DevOps/Scripts/Save-AzDevOpsBuild.ps1

```diff
@@ -1,26 +1,35 @@
 param(
    [Parameter(Mandatory = $true)][string] $ProjectId = $(throw "ProjectId is required"),
-   [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required")
+   [Parameter(Mandatory = $true)][string] $BuildId = $(throw "BuildId is required"),
+   [Parameter(Mandatory = $false)][int] $DaysToKeep
 )
 
-$retentionPayload = @{
-  keepforever='true'
+if ($DaysToKeep -eq '' -Or $DaysToKeep -eq 0) {
+    $daysValid = 99999
```

Comment:
This is the max?

Reply:
The max is probably a max int value.
According to the [docs](https://learn.microsoft.com/en-us/rest/api/azure/devops/build/leases/add?view=azure-devops-rest-7.0#newretentionlease):
> A retention lease valid for more than 100 years (36500 days) will display as retaining the build "forever".

However I tried with a value of 36501 and it did not show up as "forever", tested with 99999 and it does show up as "forever".